conf/pika.conf

```diff
@@ -1,141 +1,280 @@
+# the orders that added to this list will not be available for users except for administrator.
+# [Advice] It's recommended to add high-risk orders to this list.
+# [Format] Orders should be separated by ",". For example: FLUSHALL, SHUTDOWN, KEYS, CONFIG
+# By default, this list is empty.
 userblacklist :
-# if this option is set to 'classic', that means pika support multiple DB, in
-# this mode, option databases enable
-# if this option is set to 'sharding', that means pika support multiple Table, you
-# can specify slot num for each table, in this mode, option default-slot-num enable
-# Pika instance mode [classic | sharding]
+
+# Running Mode of Pika.
+# If this mode is set to 'classic', meas Pika support multiple DB, in this mode,
+# option databases enable.
+# If this mode is set to 'sharding', means pika support multiple Table, you can
+# specify slot num for each table, in this mode, option default-slot-num enable.
+# Pika supports two instance modes:  [classic | sharding]
 instance-mode : classic
-# Set the number of databases. The default database is DB 0, you can select
-# a different one on a per-connection basis using SELECT <dbid> where
-# dbid is a number between 0 and 'databases' - 1, limited in [1, 8]
+
+# The number of databases when Pika runs in classic mode.
+# The default database is DB 0, you can select a different one on
+# a per-connection basis using SELECT <dbid> where dbid is
+# a number between 0 and 'databases' - 1.
+# the [value range] of this parameter is [1, 8]
 databases : 1
-# default slot number each table in sharding mode
+
+# The default slot number of each table when Pika runs in sharding mode.
 default-slot-num : 1024
-# replication num defines how many followers in a single raft group, only [0, 1, 2, 3, 4] is valid
+
+# The number of  followers in a single raft group, only [0, 1, 2, 3, 4] is valid.
+# By default, this num is set to 0, which means this feature is [not enabled].
 replication-num : 0
-# consensus level defines how many confirms does leader get, before commit this log to client,
-#                 only [0, ...replicaiton-num] is valid
+
+# consensus level defines the num of confirms(ACKs) the leader node needs to receive from
+# follower nodes before returning the result to the client that sent the request.
+# The [value range] of this parameter is: [0, ...replicaiton-num].
+# Default value of consensus-level is 0, means this feature is not enabled.
 consensus-level : 0
-# Dump Prefix
+
+# The Prefix of dump file's name.
+# All the files that generated by order "bgsave" will be name with this prefix.
 dump-prefix :
-# daemonize  [yes | no]
+
+# daemonize  [yes | no].
 #daemonize : yes
-# Dump Path
+
+# The directory to stored dump files that generated by order "bgsave".
 dump-path : ./dump/
-# Expire-dump-days
+
+# Expire-time of dump files that generated by order "bgsave".
+# Any dump files that exceed this expire time will be cleaned up.
```

Comment:
TTL of the dump files generated by the command ``bgsave``(ditto.)
Any dump files which exceed this TTL will be deleted.

emm, the term "expire time" is more likely a timestamp IMO